frontend/src/SignupPage/SignupPage.jsx

```diff
@@ -99,11 +98,13 @@ class SignupPage extends React.Component {
                     {this.ssoConfigs.configs?.git?.enabled && (
                       <GitSSOLoginButton text="Sign up with GitHub" configs={this.ssoConfigs.configs?.git?.configs} />
                     )}
-                    <div className="mt-2 separator">
-                      <h2>
-                        <span>OR</span>
-                      </h2>
-                    </div>
+                    {this.ssoConfigs.configs?.git?.enabled && this.ssoConfigs.configs?.google?.enabled && (
```

Comment:
should be or condition, if google or git is enabled, or text should be shown
```suggestion
                    {this.ssoConfigs.configs?.git?.enabled || this.ssoConfigs.configs?.google?.enabled && (
```